### PR TITLE
Add support for gateway status

### PR DESCRIPTION
**Pull request by definitio**
This PR adds client for monitoring gateway(s) status(es) via API.

### CODE REVIEW

**Comment by definitio** [comment]
Tests are broken.

**Comment by mtreinish** [comment]
I just opened up #7 to update CI, I'll update there to get things working again. Then we'll just have to rebase this

File: pyopnsense/routes.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from pyopnsense import client
```
Comment (mtreinish) [inline pyopnsense/routes.py:1]: Can you add a license statement to this file (you can use your copyright line if you want) but I think we should include the same gpl header as the other modules. For example:

https://github.com/mtreinish/pyopnsense/blob/master/pyopnsense/firmware.py#L3-L16

**Review by mtreinish — COMMENTED** [review]
This LGTM, seems straightforward enough. One small housekeeping thing is the new routes module is missing the license header. Other than I think this is good to merge

**Comment by mtreinish** [comment]
#7 has merged now so if you rebase CI should be unblocked now

File: pyopnsense/routes.py (hunk 1 above)
Comment (definitio) [reply pyopnsense/routes.py:18]: Done

**Review by mtreinish — APPROVED** [review]
LGTM, I tested this locally and it works as expected. Thanks for pushing this.

**Pull request merged**